packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -16,6 +16,10 @@ import { RowParams } from './params/rowParams';
  * Set of icons used in the grid component UI.
  */
 export interface IconsOptions {
+  /**
+   * Icon displayed on the side of the column header title to display the filter input component.
+   */
+  columnFiltering?: React.ElementType;
```

Comment:
I believe that we should use the `components` API for this instead. It's meant to solve this very problem: customize which components are used. We will be able to use the same one once we migrate to the unstyled/styled pattern.

```suggestion
```